Dynatrace/dynatrace-operator
# Add livenessprobe to OneAgent from image

## Description

Similar to the readinessProbe now the livenessProbe is set by checking depending if the Healthcheck is set in the dynakube status. Here, the livenesProbe will be set only if the readinessProbe is not nil.

## How can this be tested?

Deploy the operator, apply a dynakube where oneagents spawn and check their status, if the livenessProbe is set or not.

## Checklist
